### PR TITLE
Move to WinRTC 6a382cec and WebRTC m84

### PR DESCRIPTION
**Pull request by fibann**
Win32 tests pass, TestAppUWP seems to work fine on x64 with H.264 and VP8, capture seems to work fine on HL2.

Destroying a video source will freeze the thread for a few seconds, related to https://github.com/microsoft/winrtc/issues/59; will fix soon.

### CODE REVIEW

File: libs/mrwebrtc/src/peer_connection.cpp [hunk 1]
```diff
@@ -186,9 +186,6 @@ class SessionDescObserver : public webrtc::SetSessionDescriptionObserver {
     RTC_LOG(LS_ERROR) << "Error setting session description: "
                       << error.message();
   }
-  void OnFailure(const std::string& error) override {
```
Comment (djee-ms) [inline libs/mrwebrtc/src/peer_connection.cpp:189]: Was this removed from the interface? This was quite helpful to diagnose issues.

File: tools/build/libwebrtc/mrwebrtc.ps1 [hunk 2]
```diff
@@ -149,43 +149,21 @@ function Install-GoogleRepository {
...
-        exit 1
-    }
-
-    # Install pywin32
-    # Make sure to use "python -m pip" and not "pip" to ensure that the same
-    # Python version that is used to compile is also used to install. Otherwise
-    # a user might have "pip" in its PATH for a Python version, and "python" for
-    # another one, and install with "pip install" will succeed but import with
-    # the other python version will still fail.
-    Invoke-Expression "python -m pip install pywin32"
```
Comment (djee-ms) [inline tools/build/libwebrtc/mrwebrtc.ps1:186]: `pywin32` is not needed anymore?

File: tools/build/libwebrtc/mrwebrtc.ps1 [hunk 3]
```diff
@@ -263,8 +241,14 @@ rtc_build_examples=false
 rtc_enable_protobuf=false
 rtc_enable_external_auth=false
 
+# Disable AV1, does not compile on UWP
+enable_libaom=false
+
 # Use WinRT video capturer for Windows Desktop and UWP
 rtc_win_video_capture_winrt=true
+
+# Use Media Foundation H.264 encoder
```
Comment (djee-ms) [inline tools/build/libwebrtc/mrwebrtc.ps1:250]: Encoder only?

File: libs/mrwebrtc/src/peer_connection.cpp (hunk 1 above)
Comment (fibann) [reply libs/mrwebrtc/src/peer_connection.cpp:189]: Yes, removed from the interface. the overload above should be good enough right?

File: tools/build/libwebrtc/mrwebrtc.ps1 (hunk 2 above)
Comment (fibann) [reply tools/build/libwebrtc/mrwebrtc.ps1:186]: Nope

File: tools/build/libwebrtc/mrwebrtc.ps1 (hunk 3 above)
Comment (fibann) [reply tools/build/libwebrtc/mrwebrtc.ps1:250]: Decoder too, will fix.

File: libs/mrwebrtc/src/peer_connection.cpp (hunk 1 above)
Comment (djee-ms) [reply libs/mrwebrtc/src/peer_connection.cpp:189]: Ah sorry didn't see it. Yes fine.

Comment (km4391) [reply libs/mrwebrtc/src/peer_connection.cpp:189]: Do you know the right place to post this question? Hey:

Does anyone know how to upgrade my platform of  Google Glasses WEBRTC platform running on android 8.1 and AWS Kinesis?

**Pull request merged**